frontend/controller/actions/group.js

```diff
@@ -169,6 +176,20 @@ export default (sbp('sbp/selectors/register', {
               }
             },
             data: inviteKeyP
+          },
+          {
+            id: SAKid,
+            name: '#sak',
+            purpose: ['sak'],
+            ringLevel: 0,
+            permissions: [],
+            allowedActions: [],
+            meta: {
+              private: {
+                content: SAKs
+              }
+            },
+            data: SAKp
```

Comment:
Shouldn't we make sure that the `#sak` has proper permissions before the server allows it to be registered?

I.e. verify `ringLevel: 0` and `permissions: []`?

Reply:
Possibly, although this isn't the place to check that.